scripts/hello-world-deploy.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/sh
+
+echo "-> Updating system"
+yes | pacman -Syu
```

Comment:
See my comment below...

Put this in a conditional to know what is the desired state? "all the package cache from the repositories should be up to date"? What's the thing you're doing here?